src/functions/questionnaire/create-quiz/handler.ts

```diff
@@ -0,0 +1,66 @@
+import QuestionnaireService from "src/database/services/quiz-api-service";
+import { middyfy } from "src/libs/lambda";
+import { DocumentClient } from "aws-sdk/clients/dynamodb";
+import type { QuestionnaireModel } from "src/database/schemas/questionnaire/questionnaire";
+import type { ValidatedEventAPIGatewayProxyEvent } from "src/libs/api-gateway";
+
+const dynamoDb = new DocumentClient();
+const questionnaireService = new QuestionnaireService(dynamoDb);
```

Comment:
This can be removed and you can import the service e.g.
import { questionnaireService } from "src/database/services";

As it is exported from the database/serveices/index.ts